version: '3'
services:
  sync:
    image: waisbrot/wait
    environment:
      TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"
  auth:
    image: p0bailey/docker-flask
    expose:
      - "9497"
    volumes:
      - "./auth_data:/tmp"
      - "./ssl:/usr/local/share/ssl"
    command: "python /tmp/mock-xivo-auth.py 9497"
  dird:
    image: xivo-dird-test
    ports:
      - "9489"
    volumes:
      - "../..:/usr/src/xivo-dird"
      - "./ssl:/usr/local/share/ssl"
      - "./etc/xivo-dird/conf.d/disable_service_discovery.yml:/etc/xivo-dird/conf.d/disable_service_discovery.yml"
      - "./etc/xivo-dird/conf.d/db_uri.yml:/etc/xivo-dird/conf.d/db_uri.yml"
      - "./etc/xivo-dird/conf.d/auth.yml:/etc/xivo-dird/conf.d/auth.yml"
      # - "${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/usr/local/lib/python2.7/site-packages/xivo"
  db:
    image: wazopbx/xivo-dird-db
    ports:
      - "5432"
